apps/catalogue/package.json

```diff
@@ -48,7 +48,6 @@
     "metadata-utils": "*",
     "nuxt-gtag": "3.0.1",
     "vite": "5.4.12",
-    "vue": "3.5.13",
-    "tailwind-components": "*"
```

Comment:
did you try a clean checkout? because that is where this somewhat annoying dep came from. By reversing build is that solved?